dist/swagger-ui.js

```diff
@@ -783,9 +531,9 @@ this["Handlebars"]["templates"]["param_readonly"] = Handlebars.template({"1":fun
   if (stack1 != null) { buffer += stack1; }
   return buffer + "</td>\n<td><span class=\"model-signature\"></span></td>\n";
 },"useData":true});
-this["Handlebars"]["templates"]["param_readonly_required"] = Handlebars.template({"1":function(depth0,helpers,partials,data) {
+this["Handlebars"]["templates"]["param_readonly"] = Handlebars.template({"1":function(depth0,helpers,partials,data) {
```

Comment:
I just ran `gulp` again and there are no updated files.  Here was my flow:

* Update the code
* Run `gulp`
* Run `npm test`

I committed after that.